Tutorials/CustomHamiltonian.ipynb

```diff
@@ -208,12 +208,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Next, we define the Hilbert space but without the constraint of on total magnetization ```total_sz```."
+    "Next, we define the Hilbert space as earlier, except that this time we do not impose the constraint on total magnetization ```total_sz```."
```

Comment:
Maybe we can briefly specify why we don't fix the magnetization? (maybe it's trivial though... )

Reply:
OK done in the commit